telegram/constants.py

```diff
@@ -247,7 +247,7 @@
 """
 from typing import List
 
-BOT_API_VERSION: str = '6.0'
+BOT_API_VERSION: str = '6.2'
```

Comment:
```suggestion
BOT_API_VERSION: str = '6.1'
```
the future isn't here.. yet.. 😃